### PR TITLE
Np 48099 Fetch serial publication - openapi spec and template

### PR DESCRIPTION
**Pull request by monaullah**
- Global timeout in template set to 30 (Due to ApiGateway timeout limit)
- Added `FetchSerialPublicationByIdentifierAndYearFunction` in template and openapi spec
- Removed unnecessary comments in template

### CODE REVIEW

**Review by LarsV123 — APPROVED** [review]

File: docs/openapi.yaml [hunk 1]
```diff
@@ -501,11 +501,76 @@ paths:
...
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SerialPublication'
+              examples:
+                objectExample:
+                  $ref: '#/components/examples/JournalResponseExample'
```
Comment (truhacevkir) [inline docs/openapi.yaml:554]: Possible to link multiple examples, `SeriesExample` in addition to `JournalResponseExample`.

**Review by truhacevkir — APPROVED** [review]
Looks good!

**Review by brinxmat — APPROVED** [review]

File: docs/openapi.yaml [hunk 2]
```diff
@@ -694,27 +731,27 @@ components:
...
-          description: The type of the returned object, always Series
+          enum: ['Series', 'Journal']
+          description: The type of the returned object, either Series or Journal
         name:
           $ref: '#/components/schemas/Name'
         onlineIssn:
           type: string
           pattern: '^[0-9]{4}-[0-9]{4}$'
-          description: The ISSN of the online edition of the Series
+          description: The ISSN of the online edition of the series/journal
```
[the remark this answers is not in the document]
Comment (monaullah) [reply docs/openapi.yaml:749]: Good catch! Updated

File: docs/openapi.yaml (hunk 1 above)
Comment (monaullah) [reply docs/openapi.yaml]: Done!

**Pull request merged**